File: Loadshop.DomainServices/Loadshop/Services/Data/SmartSpotPrice.cs
using System;

namespace Loadshop.DomainServices.Loadshop.Services.Data
{
    public class SmartSpotPrice
    {
        public Guid LoadId { get; set; }
        public decimal Price { get; set; }
        public decimal? DATGuardRate { get; set; }
        public decimal? MachineLearningRate { get; set; }
    }
}

File: Loadshop.Web.UI/src/app/shared/models/smart-spot-price.ts
export interface SmartSpotPrice {
    price: number;
    loadId: string;
    datGuardRate: number;
    machineLearningRate: number;
    loading: boolean;
}

export const defaultSmartSpotPrice: SmartSpotPrice = {
    price: 0.00,
    loadId: null,
    datGuardRate: null,
    machineLearningRate: null,
    loading: true
};
